config/config.go

```diff
@@ -97,6 +97,13 @@ var (
 	DefaultServersetSDConfig = ServersetSDConfig{
 		Timeout: Duration(10 * time.Second),
 	}
+
+	// DefaultMarathonSDConfig is the default Marathon SD configuration.
+	DefaultMarathonSDConfig = MarathonSDConfig{
+		Name:           "marathon",
+		UseJobPrefix:   false,
+		UpdateInterval: Duration(10 * time.Second),
```

Comment:
We usually default to 30s for polling as anything faster has little effect considering that scrape intervals are rarely more frequent.